### PR TITLE
Add rust schema

### PR DESCRIPTION
**Pull request by GreenBayRules**
data/Language/Rust.yaml >> new Rust language schema + doc_comment delimeter yaml

### CODE REVIEW

File: data/Language/Rust.yaml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+ - impl
+ - ref
+ - return
+ - Self
+ - self
+ - static
+ - struct
+ - super
+ - trait
+ - true
```
Comment (jayvdb) [inline data/Language/Rust.yaml]: true & false are a boolean datatype .  Please see other recently added languages for how to do this

Comment (GreenBayRules) [reply data/Language/Rust.yaml]: fixed

File: data/Language/Rust.yaml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+ - break
+ - const
+ - continue
+ - crate
+ - else
+ - if
+ - if let
+ - enum
+ - extern
+ - 'false'
```
Comment (jayvdb) [inline data/Language/Rust.yaml]: remove false -- it is one of the boolean literals, so we dont need it here

File: data/Language/Rust.yaml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+ - impl
+ - ref
+ - return
+ - Self
+ - self
+ - static
+ - struct
+ - super
+ - trait
+ - 'true'
```
Comment (jayvdb) [inline data/Language/Rust.yaml]: remove

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being rebased automatically. Please **DO NOT** push while rebase is in progress or your changes would be lost permanently :warning:

**Comment by jayvdb** [comment]
Automated rebase with [GitMate.io](https://gitmate.io) was successful! :tada:

File: data/Language/Rust.yaml [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+  - doc_comment
+  - multiline_slash_star
+  # string delimiters:
+  - c_char
+  - double_quote_slash_escape
+  - curly_braces
+datatypes:
+  - boolean_rust
+keywords:
+ - as
```
Comment (jayvdb) [inline data/Language/Rust.yaml]: wrong indentation: expected 2 but found 1 (indentation)

*Origin: YAMLLintBear, Section: `yaml`.*

Comment (jayvdb) [reply data/Language/Rust.yaml]: The bot to the rescue.  I'll fix the indentation

**Comment by jayvdb** [comment]
ack https://github.com/coala/coAST/pull/35/commits/2ef4b2c7dc2228eadda89bcb403bfb15f5c73d4c

**Comment by jayvdb** [comment]
https://github.com/coala/coAST/pull/35/commits/cf9269b5d865c462038f746c857cb220e46ed96fack

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being fastforwarded automatically. Please **DO NOT** push while fastforward is in progress or your changes would be lost permanently :warning:

**Comment by jayvdb** [comment]
Automated fastforward with [GitMate.io](https://gitmate.io) was successful! :tada:

**Pull request merged**